### ISSUE TITLE
Use log_softmax twice?

### ISSUE THREAD

**Issue by RaptorMai** [paraschopra/bayesian-neural-network-mnist#5]
Hi,
In model(x_data, y_data), lhat = log_softmax(lifted_reg_model(x_data)).

In give_uncertainities(x), yhats = [F.log_softmax(model(x.view(-1,28*28)).data, 1).detach().numpy() for model in sampled_models]

Does it mean you use log_softmax twice?
